### PR TITLE
Removing help message from error message

### PR DESCRIPTION
**Pull request by chshrikh**

### CODE REVIEW

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 [hunk 1]
```diff
@@ -116,15 +116,6 @@ function Write-TaskSpecificTelemetry
     Write-Telemetry "$codeKey" "3B5693D4-5777-4FEE-862A-BD2B7A374C68"
 }
 
-function ThrowError
```
Comment (kmkumaran) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: Do we have this in any other task?

**Comment by pryenugu** [comment]
Windows Machine copy and Azure File copy tasks also have throw-Error, are they applicable?

File: Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1 (hunk 1 above)
Comment (chshrikh) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: windows file copy ( robocopy ) we need to remove

Comment (chshrikh) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: I am not sure for azure file copy

Comment (chshrikh) [inline Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1]: updated for windows file copy . For azure file copy we can check with PM and update it

**Comment by kmkumaran** [comment]
Looks fine. Are there any Pester test required to change looking for this message?

**Pull request merged**